features/cellular/framework/targets/UBLOX/AT/UBLOX_AT_CellularStack.cpp

```diff
@@ -449,13 +449,14 @@ nsapi_error_t UBLOX_AT_CellularStack::gethostbyname(const char *host, SocketAddr
     if (address->set_ip_address(host)) {
         err = NSAPI_ERROR_OK;
     } else {
-        // This interrogation can sometimes take longer than the usual 8 seconds
-        _at.cmd_start_stop("+UDNSRN", "=0,", "%s", host);
 #ifdef TARGET_UBLOX_C030_R41XM
         _at.set_at_timeout(70000);
```

Comment:
When exiting this method restore the timeout value, If there is a reason that not done already

Reply:
Already present at the end. [restore_at_timeout](https://github.com/u-blox/mbed-os/blob/b2d0448351aac4910fbb667f7d511c850460c2e3/features/cellular/framework/targets/UBLOX/AT/UBLOX_AT_CellularStack.cpp#L469)